docs/releases/0.4-NOTES.md

```diff
@@ -0,0 +1,19 @@
+## Release notes for poseidon v0.4
+
+# Significant changes
+Node Taints & Toleration Feature.
+
+# Required Actions
+None
+
+# Highlighted changes
+* [Adding Taints & Toleration Feature, few performance improvements and minor bug fixes](https://github.com/kubernetes-sigs/poseidon/pull/128)
+
```

Comment:
What about adding one section that describes the performance improvements?

Reply:
We successfully resolved the negative impact of pod level anti-affinity symmetry on the normal pods. For normal pods with no pod level anti-affinity symmetry conflict, we see very substantial benefits as we observed in the earlier throughput performance testing exercise we did (20-30X better).

With the latest design changes, irrespective of size of the "Jobs" (Replicaset, Deployment or Jobs), we are seeing substantial throughput benefits using Firmament scheduler as long as resource requirements (CPU/Mem) for incoming Pods are uniform. This is mainly because Firmament is able to efficiently amortize work across ReplicaSets/Deployments/Jobs.

Reply:
@hanxiaoshuai Please update it.

Reply:
@m1093782566 Done